### ISSUE TITLE
Accidentally-removed lessons

### ISSUE THREAD

**Issue by mdlincoln** [programminghistorian/jekyll#468]
It looks like somewhere during the merging of @amandavisconti changes, several lessons were erased from @fredgibbs branch. I'm going to try and figure out the right bit of filtering and commit reverting to fix that and send them back up to Fred via PR. I'll keep notes in this issue.

**Comment by mdlincoln** [comment]
From the offending commit f414cba5abfd6942cd9bdcd98b4babda51d2d44d

```
lessons/basic-text-processing-in-r
lessons/geocoding-qgis.md
lessons/intro-to-linked-data.md
lessons/text-mining-with-extracted-features.md
lessons/using-javascript-to-create-maps.md
```

**Comment by acrymble** [comment]
I think those are all lessons published since that work initially began.

**Comment by mdlincoln** [comment]
That's correct. Because the lesson page redesign was started from scratch, instead of by making a github fork of the repository, it wasn't possible to do a `git merge` operation that would have recognized that these newer lessons should not have been _deleted_ as part of the merge. In the future, we will be careful to do any major development work like this through branches/forks, rather than making new copies of the files in the site.

Luckily, it's just a matter of reverting the changes to those affected lesson files - I've already submitted a PR to Fred. I also should have caught this issue back when we first did the merge in to Fred's fork... but now I know what to look out for.
